script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -212,7 +212,7 @@ Clouds have begun to gather above us,
 -- Page 11, Offset 41418.
 -- 神父服の男がどんな<表|か><情|お>をしているのか、やはり読み取れない。
 // 神父服の feels a little clumsy here. Maybe we just shorted it to priest?
-I still really can't tell what kind of expression is on the priest's face.
+The priest's expression remains indistinguishable.
```

Comment:
Indistinguishable makes it sound like he can see it but not interpret it, but I think earlier he comments that he just can't get a clear look at his face at all?

> I still can't get a good look at the expression on the priest's face.

Reply:
It seems more like it's sort of blurry or indistinct
I think the original is fine